_dev/js/back/src/components/panel/express-checkout.vue

```diff
@@ -47,16 +43,18 @@
                   v-if="orderPageIsActive"
                   class="active-img mb-2"
                   src="@/assets/images/preview_cart-page_active.png"
-                >
+                  alt=""
```

Comment:
I know it's on the BO, but why adding an empty alt, it almost the same than not adding it except that it's useless :p

Reply:
Just because eslint say alt attribute is required... 🤷‍♂️

Reply:
Then I think we should fill them, and it's doable :

put an i18n chain : `Active page preview icon` for this one should do the job